src/components/PreviewComment/index.js

```diff
@@ -43,29 +43,22 @@ const PreviewComment = ({comment, time, image, totalComment, onPress, user, item
           <View style={{left: -16}} />
           <View style={styles.profile}>
             {item?.data?.anon_user_info_emoji_name || item?.data?.is_anonymous ? (
-              <View style={[styles.image, {backgroundColor: item.data.anon_user_info_color_code}]}>
-                <Text style={{color: COLORS.white, fontSize: 14}}>
-                  {item?.data?.anon_user_info_emoji_code}
-                </Text>
-              </View>
+              <ProfilePicture
+                karmaScore={item.data.karmaScores}
+                size={25}
+                width={6}
+                withKarma
+                isAnon={true}
+                anonBackgroundColor={item.data.anon_user_info_color_code}
+                anonEmojiCode={item?.data?.anon_user_info_emoji_code}
+              />
             ) : (
-              <Image
-                source={
-                  image
-                    ? {uri: image, cache: 'reload'}
-                    : require('../../assets/images/ProfileDefault.png')
-                }
-                loadingIndicatorSource={
-                  <Image
-                    style={{
-                      width: 24,
-                      height: 24,
-                      backgroundColor: COLORS.white,
-                      borderRadius: 12
-                    }}
-                  />
-                }
-                style={styles.image}
+              <ProfilePicture
+                karmaScore={item.karmaScores}
+                profilePicPath={image || require('../../assets/images/ProfileDefault.png')}
+                size={25}
+                width={6}
+                withKarma
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of the `ProfilePicture` component here is a significant improvement over the previous implementation. It simplifies the code by abstracting the logic of handling anonymous and regular users into a separate component. This makes the code more readable and maintainable. However, there's a discrepancy in the prop `karmaScore`. For anonymous users, it's `item.data.karmaScores`, but for regular users, it's `item.karmaScores`. Ensure that this is intentional and not a mistake.
```diff
- karmaScore={item.karmaScores}
+ karmaScore={item?.data?.karmaScores || item?.karmaScores}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->